tests/quantization/bnb/test_mixed_int8.py

```diff
@@ -833,4 +832,4 @@ def test_int8_from_pretrained(self):
         encoded_input = self.tokenizer(self.input_text, return_tensors="pt")
         output_sequences = model.generate(input_ids=encoded_input["input_ids"].to(0), max_new_tokens=10)
 
-        self.assertEqual(self.tokenizer.decode(output_sequences[0], skip_special_tokens=True), self.EXPECTED_OUTPUT)
+        self.assertIn(self.tokenizer.decode(output_sequences[0], skip_special_tokens=True), self.EXPECTED_OUTPUTS)
```

Comment:
I observed that the tests outputs vary depending on the GPU architecture used.  I have A100s and the testing system has something else. 
I see the need for uniform approach here (single vs multiple vs explicitly driven by GPU type correct outputs). It is really up to the maintainers. Also this can be changed in a separate PR.

Just let me keep these multiple outputs for now - they are based on the resutls that come from the main branch, and help me focus on the failing tests.